test.js

```diff
@@ -1,19 +1,19 @@
 
 
-var numberOfStudentsEjected = 0;
+var numberOfStudentsRemove = 0;
 
 function sayHi() {
   alert("Hi2");
   alert("Hi3");
 }
 
-function ejectStudent() {
+function removeStudent() {
   // Update the database
-  numberOfStudentsEjected += 1;
-  displayNumberOfStudentsEjected();
+  numberOfStudentsRemove += 1;
+  displayNumberOfStudentsRemoved();
```

Comment:
test(pls ignore me)

Reply:
Ah yes, this feature would be great, but way out of scope for this PR. Consider yourself ignored.